astro/src/content/blog/passwordless-authentication.mdx

```diff
@@ -0,0 +1,160 @@
+---
+publish_date: 2023-11-10
+title: The Security Implications Of Passwordless Authentication
+description: Learn about passwordless authentication
+authors: Dan Moore
```

Comment:
Don't care who this is, happy to have it ghostwritten. Also happy to have it be one of Ritza authors.